README.md

```diff
@@ -244,6 +244,10 @@ Alternatively, you can execute commands from the MediaWiki root directory:
 
 ## Release notes
 
+### Version 3.0.0 - 2025-02-25
+
+* Bump minimum MediaWiki requirements to 1.39 
```

Comment:
If this is all we did, it does not warrant a release, because there is nothing new. So while bumping the version makes sense, tagging now does not.

Also, we write release notes in past tense